pkg/transformer/openshift/openshift.go

```diff
@@ -197,13 +191,18 @@ func (o *OpenShift) initImageStream(name string, service kobject.ServiceConfig,
 	return is
 }
 
-// initBuildConfig initialize Openshifts BuildConfig Object
 func initBuildConfig(name string, service kobject.ServiceConfig, repo string, branch string) (*buildapi.BuildConfig, error) {
+	image := name
```

Comment:
Could you describe "same as above"?. GitHub does not show your other comments / splits them apart.